React-Homeworks/src/components/FoodCard/FoodCard.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import "./FoodCard.css";
+import Button from "../Buttons/Buttons";
+
+function FoodCard({ title, subTitle, price, imgUrl }) {
+  const [quantity, setQuantity] = useState("1"); 
+
+  const handleChange = (event) => {
+    const newQuantity = event.target.value;
+    setQuantity(newQuantity);
+  };
+
+  return (
+    <figure className="card flex-elem">
```

Comment:
I'm not sure the tag figure is suitable as a wrapper for the card component. Because it's not an image with some caption; it's a multi-functional block.